docker/launch_spark_history_server_locally.sh

```diff
@@ -1,20 +1,92 @@
-#!/bin/sh
+#!/bin/sh 
+set -e
 # ENTER AWS CREDENTIALS TO RUN THE DOCKER IMAGE LOCALLY
 
-DOCKER_IMAGE="varabonthu/spark-web-ui:1.0.6"
-LOG_DIR="s3a://<bucket_name>/<spark-event-logs-prefix>/"
-ENDPOINT="s3.<region>.amazonaws.com"
+CLASS="org.apache.spark.deploy.history.HistoryServer"
+AWS_REGION="${4:-us-east-1}"
```

Comment:
Should this be as `$3`?  Like AWS_REGION="${3:-us-east-1}"

Reply:
No, as per the command its the 4th argument after the action(start/stop) bucket and prefix.